src/webapi/date.rs

```diff
@@ -1,25 +1,1166 @@
-use std::marker::PhantomData;
-use webcore::try_from::TryInto;
+use std::collections::HashMap;
+use webcore::try_from::{TryFrom, TryInto};
+use webcore::value::{Value, Reference};
 
-/// [(JavaScript docs)](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Date)
-#[derive(Debug)]
-pub struct Date {
-    dummy: PhantomData< () >
-}
-
-impl Date {
-    /// The Date.now() method returns the number of milliseconds elapsed since 1 January 1970 00:00:00 UTC.
+/// A date object
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Date)
```

Comment:
Broken link.

Reply:
sorry about that, thanks for the catch! Same with the (highly experimental) ecosystem stuff - accidentally polluted this PR